fetch.bs

```diff
@@ -3898,20 +4116,31 @@ these steps:
 
  <li><p>Let <var>actualResponse</var> be null.
 
+ <li><p>Let <var>timingInfo</var> be <var>fetchParams</var>'s<a for="fetch params">timing
```

Comment:
Editorial nit: missing space between fetchParams and the next word.
Also, no need for "'s" after fetchParams here and elsewhere, as "'" would be sufficient

Reply:
For the `'s` - it's true for many existing parts in  fetch spec, wanted to stay consistent.

Reply:
OK, that's fair